app/src/features/city/subsectorSlice.ts

```diff
@@ -2,111 +2,34 @@ import { RootState } from "@/lib/store";
+}
+
 
 interface SubsectorState {
   subsector?: SubSectorAttributes;
-  subCategories: SubCategoryAttributes[];
-  scopes: ActivityDataScope[];
+  manualInputSchema: any
 }
 
 const initialState = {
   subsector: undefined,
-  subCategories: [],
-  scopes: [
-    {
-      scope: 1,
-      formInputs: SectorFormSchema,
-      methodologies: [
-        {
-          methodologyId: "1",
-          methodologyName: "Fuel Combustion Consumption",
-          description:
-            "Direct recording of fuels combusted in commercial buildings",
-          inputRequired: ["Total fuel consumed amount"],
-          disabled: false,
-        },
-        {
-          methodologyId: "2",
-          methodologyName: "Scaled sample data",
-          description:
-            "Extrapolates emissions from a representative sample of buildings.",
-          inputRequired: [
-            "Sample fuel consumed amount",
-            "Scaling data (population, GDP, area, etc.) for sample and city level",
-          ],
-          disabled: true,
-        },
-        {
-          methodologyId: "3",
-          methodologyName: "Modeled data",
-          description: "Emissions estimated from predictive models.",
-          inputRequired: ["Modeled fuel intensity consumption", "Build area"],
-          disabled: true,
-        },
-        {
-          methodologyId: "4",
-          methodologyName: "Direct measure",
-          description:
-            "Direct emission measurements from commercial buildings' combustion sources.",
-          inputRequired: ["Emissions data"],
-          disabled: false,
-        },
-      ],
-    },
-    {
-      scope: 2,
-      formInputs: SectorFormSchema,
-      methodologies: [
-        {
-          methodologyId: "1",
-          methodologyName: "Energy Consumption",
-          description:
-            "Direct recording of fuels combusted in commercial buildings",
-        },
-        {
-          methodologyId: "2",
-          methodologyName: "Scaled sample data",
-          description:
-            "Extrapolates emissions from a representative sample of buildings.",
-          inputRequired: [
-            "Sample fuel consumed amount",
-            "Scaling data (population, GDP, area, etc.) for sample and city level",
-          ],
-          disabled: true,
-        },
-        {
-          methodologyId: "3",
-          methodologyName: "Modeled data",
-          description: "Emissions estimated from predictive models.",
-          inputRequired: ["Modeled fuel intensity consumption", "Build area"],
-          disabled: true,
-        },
-        {
-          methodologyId: "4",
-          methodologyName: "Direct measure",
-          description:
-            "Direct emission measurements from commercial buildings' combustion sources.",
-          inputRequired: ["Emissions data"],
-          disabled: false,
-        },
-      ],
-    },
-  ],
+  manualInputSchema: manualInputHierarchySchema
```

Comment:
Not quite sure why we need to store this static JSON file in Redux, since presumably it won't change?